app/schema.graphql

```diff
@@ -330,7 +331,7 @@ type Mutation {
     subscription: String!
     session: String
     viewName: String!
-    datasetName: String
+    datasetName: String = null
```

Comment:
Shouldn't this be required?

Reply:
I didn't change this. The schema is autogenerated from the code. Do you want me to change the mutation?

Reply:
right now it's technically not required input because we can just read it from the state

Reply:
I'm just confused why this is appearing in this PR

Reply:
This should be required. We shouldn't assume state in mutations.

Reply:
ok I'll make it required

@brimoor With no code change, the autogenerated schema would only be different if someone manually changed this file or changed the mutation without regenerating the schema